app/controllers/api/v1/cart_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Api::V1::CartController < ApplicationController
+  load_and_authorize_resource :cart
```

Comment:
are you actually using this? Cart isn't technically a resource so I don't think this would work.